Uplift/Controllers/SportsFeedViewController.swift

```diff
@@ -62,24 +74,41 @@ class SportsFeedViewController: UIViewController {
         calendarCollectionView.register(CalendarCell.self, forCellWithReuseIdentifier: CalendarGenerator.calendarCellIdentifier)
         calendarCollectionView.layer.zPosition = -1
         view.addSubview(calendarCollectionView)
-        
+
         let layout = UICollectionViewFlowLayout()
         layout.minimumLineSpacing = 12.0
-        layout.itemSize = CGSize(width: UIScreen.main.bounds.width - 32.0, height: 100.0)
+        layout.itemSize = CGSize(width: UIScreen.main.bounds.width - 32.0, height: PickupGameCell.height)
         layout.sectionInset = .init(top: 0.0, left: 16.0, bottom: 0.0, right: 16.0)
-        
+
         collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
         collectionView.backgroundColor = .white
         collectionView.dataSource = self
         collectionView.delegate = self
         collectionView.showsVerticalScrollIndicator = false
-        collectionView.register(PickupGameCell.self, forCellWithReuseIdentifier: sportIdentifier)
+        collectionView.register(PickupGameCell.self, forCellWithReuseIdentifier: Identifiers.pickupGameCell)
         view.addSubview(collectionView)
-        
+
+        profileView.myGames = posts[3]
+        profileView.joinedGames = posts[3]
+        profileView.pastGames = posts[3]
+
+        sideMenu = SideMenuNavigationController(rootViewController: ProfileViewController(from: profileView))
```

Comment:
this seems kind of odd to me. why are we passing in the profile view here instead of just instantiating it inside the `ProfileViewController`?

Reply:
My reasoning was that we needed access to it in `SportsFeedViewController` since we set the profile view's games there, but I can make it so that `ProfileViewController` takes in the games as an array and then passes them to the profile view.